QuadPaySDK/PaymentWidget/PaymentWidget.swift

```diff
@@ -75,6 +75,45 @@ public final class PaymentWidget: UIView {
     
     override public init(frame: CGRect) {
         super.init(frame: .zero)
+        UIFont.jbs_registerFont(
```

Comment:
For the fonts can we not either
1. Load the fonts when the library is loaded.
2. Or create an extension method which registers all these fonts (since the same fonts are loaded in the `RNPaymentWidget` component)